poc/vidpf.py

```diff
@@ -86,16 +89,10 @@ def gen(cls, alpha, beta, binder, rand):
                 w_cw[j] *= mask
 
             # Compute hashes for level i
-            sha3 = hashlib.sha3_256()
-            sha3.update(str(node).encode('ascii') +
-                        to_le_bytes(i, 2) + seed[0])
-            proof_0 = sha3.digest()
-            sha3 = hashlib.sha3_256()
-            sha3.update(str(node).encode('ascii') +
-                        to_le_bytes(i, 2) + seed[1])
-            proof_1 = sha3.digest()
-
-            cs_proofs.append(xor(proof_0, proof_1))
+            cs_proofs.append(xor(
+                next_cs_proof(i, seed[0]),
```

Comment:
I think the prefix/node is getting left out here.

Reply:
Good catch, I completely missed this. Modified `next_cs_proofI()` to take in the node as input. Please double check how the node is encoded.